vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -90,39 +90,67 @@ The meaning of each of the detail arguments is as follows:
 
```

Comment:
I would actually prefer to not explicitly put the defaults and allowed values in writing.. They change from time to time and thus we would always have to make sure to update both the single source of truth (stress_test_arguments) and the vignette. Also the information is already here https://github.com/2DegreesInvesting/r2dii.climate.stress.test/blob/bf47f3895807654944f8334fc43bb38247e01fda/vignettes/articles/03-run-stress-test.Rmd#L85 
if you think it would be better to mention the defaults and possible values directly in the definitions we should programmatically get the value from stress_test_arguements, I can write a small helper function fot that.
On scenario_geographies the way it is presented here it is not apparent that the scenario_geographies are usually not fully supported but only for some sectors, so it might create unrealistic expectations. In that regard I would argue the explanation here is more adequate: https://github.com/2DegreesInvesting/r2dii.climate.stress.test/blob/bf47f3895807654944f8334fc43bb38247e01fda/vignettes/articles/02-prepare-loans-inputs.Rmd#L54

Reply:
I take your advice. The list of geographies is now included in 02-prepare-loans-inputs.Rmd

I still think we should state the default values in the vignettes either in the list of input parameters or close and yes a programatic way of updating it would be great. I suggest we do this in a separate PR